tools/accuracy_checker/openvino/tools/accuracy_checker/preprocessor/launcher_preprocessing/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+Copyright (c) 2018-2022 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from .ie_preprocessor import IEPreprocessor, ie_preprocess_available
```

Comment:
No module named 'openvino.tools.accuracy_checker.preprocessor.ie_preprocessor' in tests